tests/_internal/windows/test_windows_identifier.py

```diff
@@ -0,0 +1,83 @@
+import unittest
```

Comment:
It would help me to see some comments around the tests as they define the behaviour.